src/content/github/pr/main.tsx

```diff
@@ -47,7 +47,12 @@ async function execute() {
   createContainer();
 
   const coverageReport = await getPRReport(urlMetadata);
-  if (!coverageReport.files) {
+  if (
+    !coverageReport.files ||
+    !coverageReport.totals.base.coverage ||
```

Comment:
just checking, should we do a null check like `coverageReport?.totals?.base?.coverage` for these values?

Reply:
oh yea you right lol